Client/Teztez/Flows/Common/ActivitiesIntro/Controller/ActivitiesIntroViewController.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  ActivitiesIntroViewController.swift
+//  Teztez
+//
+//  Created by Adlet on 4/23/20.
+//  Copyright © 2020 crococoders. All rights reserved.
+//
+
+import UIKit
+
+protocol ActivitiesIntroPresentable: Presentable {
+    var onNextDidTap: Callback? { get set }
+}
+
+final class ActivitiesIntroViewController: UIViewController, ActivitiesIntroPresentable {
+    var onNextDidTap: Callback?
+
+    private let store: ActivitiesIntroStore
+
+    @IBOutlet private var titleImageView: UIImageView!
+    @IBOutlet private var titleLabel: UILabel!
+    @IBOutlet private var nextButton: CommonButton!
+
+    @IBOutlet private var infoImageView: UIImageView!
+    @IBOutlet private var infoTitleLabel: UILabel!
+    @IBOutlet private var infoDescriptionLabel: UILabel!
+
+    @IBOutlet private var usageImageView: UIImageView!
+    @IBOutlet private var usageTitleLabel: UILabel!
+    @IBOutlet private var usageDescriptionLabel: UILabel!
+
+    init(store: ActivitiesIntroStore) {
```

Comment:
So that you can just pass `ActivitiesIntroViewModel` here. I think it's enough for now.